Formula/l/lume.rb

```diff
@@ -0,0 +1,21 @@
+class Lume < Formula
+  desc "Create and manage Apple Silicon-native virtual machines"
+  homepage "https://github.com/trycua/lume"
+  url "https://github.com/trycua/lume/archive/refs/tags/v0.1.6.tar.gz"
+  sha256 "4f3216eef9c2a805cbbd3893efef3e161e00ad76718a00623c0481d2c2855767"
+  license "MIT"
+  head "https://github.com/trycua/lume.git", branch: "main"
+
+  uses_from_macos "swift" => :build
+  depends_on :macos
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release", "--product", "lume"
+    bin.install ".build/release/lume"
+  end
+
+  test do
+    system bin/"lume", "ipsw"
```

Comment:
Probably because of the heavy network requirement here we could strip it down to some simpler functionality like `lume ls`, `lume serve`. If you grep the homebrew-core repo for `free_port` there's other examples of formula which do functional tests against server functionality.

Reply:
Probably more functional stuff to do with actual VM launching might be hindered by CI's inability to do nested virtualization